website/_i18n/ar/user-guides/run-pactus-metrics/index.md

```diff
@@ -0,0 +1,136 @@
+---
+version: 1
+---
+
+## Preface
+
+Pactus can be run with metrics, providing you with the ability to monitor your node.
+This tutorial will guide you through the steps to run Pactus with metrics, suitable for advanced users.
+
+## Prerequisites
+
+Before proceeding with the steps below, ensure that you have the following:
+
+- An understanding of Prometheus and Grafana.
+- Have a running Pactus Node.
+
+## Configure Pactus Node for Metrics
+
+To Configure Pactus Node for Metrics,navigate to the Pactus directory; by default, it’s located at the following path.
+
+```text
+/home/YourUsername/pactus
+```
+
+there’s a file named `config.toml` that contains all the configurations for your node.
+There are two parameters that you should enable for metrics: the first is `enable_metrics`,
+and the second is enable `http`. After editing the `config.toml` you should restart your node.
+
```

Comment:
The metrics now can be accessed at [http://localhost:80/metrics/prometheus](http://localhost/metrics/prometheus) (this url going to be use by prometheus).

{% alert_start warning %}

if you are running Pactus with docker image, make sure to expose :80 port.

{% alert_end %}